eventHandler.js

```diff
@@ -0,0 +1,34 @@
+var CarLot = (function (oldCarLot) {
+// grab all necessary DOM elements for event listeners
+  var descriptionInput = document.getElementById('descriptionInput'),
+      submitButton = document.getElementById('descriptionSubmit'),
+      productCards = document.getElementsByClassName('carProductCard');
+
+// function to encapsulate the adding of event listeners to all DOM elements
+  oldCarLot.activateEvents = function () {
+  	// event listener for keydown press in text input box for adding description
+  	// to car product cards
+    	descriptionInput.addEventListener('keyup', function (event) {
+    		for (var i = 0, j = productCards.length; i < j; i++) {
+    			let editDescription = productCards[i];
+    			// check for the 'active' (clicked) card
+    			if (editDescription.classList.contains('active')) {
+    				descriptionOutput = editDescription.querySelector(`#carDescription${i}`);
+    				// insert input value to selected card description
+    				descriptionOutput.innerText = descriptionInput.value;
+    			}
+    		}
+    	});
+    // loop over dynamically created product cards and add event listeners
+    	for (var i = 0, j = productCards.length; i < j; i++) {
+    		let productCard = productCards[i];
+    		productCard.addEventListener('click', function (clickEvent) {
+    			// pass click event to callback function
+    			CarLot.addStyles(clickEvent);
```

Comment:
When you defined the addStyles function you used it to pass the requirement "The function must accept two arguments. 1 car DOM element that was clicked on. 2 color name." So you defined it to accept a color but never pass it a color. 
Try creating a separate function or variable to get the color of the car clicked on and pass it in.

All in all, looks nice though!